Micromouse/Mouse.cpp

```diff
@@ -44,6 +44,10 @@ void Mouse::reset() {
     // TODO
 }
 
+Maze &Mouse::getMaze() {
```

Comment:
If you provide read/write access, does this have any advantage over just making the member public?